packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -1,31 +1,10 @@
 @import './variables';
 
-:root {
-  // Body
-  --terra-consumer-body-background-image: url('../background.jpg');
-  --terra-consumer-body-background-color: #c7d4ea;
-
-  // Nav
-  --terra-consumer-nav-background-color: rgba(255, 255, 255, 0.25);
-  --terra-consumer-nav-width: 320px;
-  --terra-consumer-nav-link-color: #3d3d3d;
```

Comment:
can we default all of the nav text color to this instead of the white that's currently there. I think its just nav text, profile text and possibly the hamburger icon as well

Reply:
updated

Reply:
looks like two more places:
https://github.com/cerner/terra-consumer/blob/a4c131631806907b672f188f1f3f79ddc5abad53/packages/terra-consumer-nav/src/components/nav-toggler/NavToggler.scss#L7
https://github.com/cerner/terra-consumer/blob/a4c131631806907b672f188f1f3f79ddc5abad53/packages/terra-consumer-nav/src/Nav.scss#L49

also, badge background should remain white:
https://github.com/cerner/terra-consumer/blob/a4c131631806907b672f188f1f3f79ddc5abad53/packages/terra-consumer-nav/src/components/nav-items/NavItem.scss#L44
<img width="309" alt="image" src="https://user-images.githubusercontent.com/30411845/30552570-6fc9d464-9c6c-11e7-8289-193062ccdf83.png">